src/components/Integrations/BigQuery/AuthorizationPage.js

```diff
@@ -59,23 +60,118 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient, dopplerUserApiCli
       <Promotional
```

Comment:
the promotional logo is very big. 
![image](https://user-images.githubusercontent.com/84402180/132217353-ec67e621-ecfb-48b8-80f9-55b7f93b7900.png)

Reply:
promo page logo and gif can be moved to BigQuery folder

Reply:
in figma you could download the logo with the original size

Reply:
you can move to outside bigquery_logo.png and google-data-studio.gif. The style folder must contain the styles, but is a empty folder. This folder should be deleted.
![image](https://user-images.githubusercontent.com/84402180/132359547-38f76d6a-800e-4a4f-948f-351ac7d9bda7.png)